content/en/functions/images/Mask.md

```diff
@@ -17,15 +17,44 @@ toc: true
 
 The `images.Mask` filter applies a mask to an image. Black pixels in the mask make the corresponding areas of the base image transparent, while white pixels keep them opaque. Color images are converted to grayscale for masking purposes. The mask is automatically resized to match the dimensions of the base image.
 
+{{% note %}}
+Of the formats supported by Hugo's imaging pipelie, only PNG and WebP have an alpha channel to support transparency. If your source image uses a different format, convert it to either PNG or WebP before applying the mask as shown in the example below.
+
+Applying an image mask to a non-transparent image format such as JPEG will result in masked areas appearing white instead of transparent.
```

Comment:
The above isn't entirely try:

* The JPEG will be filled with the `bgColor` defined in `hugo.toml` (default white)
* If you combine the mask with e.g. `images.Process "#000"` the background will be black.